pkg/apis/core/v1/defaults.go

```diff
@@ -216,6 +217,11 @@ func SetDefaults_Pod(obj *v1.Pod) {
 			}
 		}
 	}
+
+	if utilfeature.DefaultFeatureGate.Enabled(features.PodLevelResources) {
```

Comment:
Maybe put this higher up, right after "set requests to limits if requests are not specified, but limits are" -- this HAS TO run _after_ that (to get all container requests) and having them far apart makes it less obvious.  I would also add a comment about per-container must come first.

Reply:
moved and added comment

Reply:
I don't see it moved?  I thought ~184 is wherre it should go